pkg/controller/controller.go

```diff
@@ -74,7 +74,12 @@ func RegisterInformers(c *config.Config) {
 			if !ok {
 				continue
 			}
-			for _, ns := range r.Namespaces {
+
+			// Populate namespaces from the config file for adding resource informers
+			namespacesToWatch := utils.PopulateNamespacesForResources(r.Namespaces.Include, r.Namespaces.Ignore)
```

Comment:
@aananthraj could you please test if following scenario work?
- start BotKube with include=all and ignore={some namespaces}
- create new namespaces e.g test2
- create resources in the namespace and verify BotKube gives expected notification

I don't think this will work as we are setting informers on particular namespaces only. For "all" namespaces, we should start informers on `apiV1.NamespaceAll` and filter notifications at `sendMessage()` or `notifier.SendMessage()`

Reply:
Yes @PrasadG193 , I missed this one. 
May be we could create a `namespace exclusion filter` for handling this one.

Reply:
@aananthraj please hold on this for now. We need to change the way we filter namespaces after introducing SharedInformerFactory (https://github.com/infracloudio/botkube/issues/143). Let's hold on this till #143 gets fixed

Reply:
Sure, lets wait then

Reply:
will work on this and update here

Reply:
updated the changes, let's get this reviewed.